[BUG] White screen after entering the 2FA Authentication code

<!-- 

Please see our guide for opening issues: https://rocket.chat/docs/contributing/reporting-issues

If you have questions or are looking for help/support please see: https://rocket.chat/docs/getting-support

If you are experiencing a bug please search our issues to be sure it is not already present: https://github.com/RocketChat/Rocket.Chat/issues

-->

### Description:

After the new registration of a user, he receives an authentication code for the activation of the 2FA by email.
The user has been checked and confirmed by the admin and the admin has specified a user name for the user.
After entering the authentication code and clicking on check, only a white page is displayed in the browser.
It is not possible to set up a new user!

The following error is displayed in the console.

![image](https://user-images.githubusercontent.com/66670106/104320178-490f1780-54e2-11eb-8343-f9f47a26b750.png)

Tested with Chrom, FireFox and Opera



### Steps to reproduce:

1) a new user registers with email and password
2) an admin defines the user name as well as activates and confirms the user
3) the user logs on to RC and then receives a mail with the authentication code
4) the user enters the authentication code in the verification form and confirms it
5) a white browser window will appear

### Expected behavior:

Next, an authentication app should be set up and a QR code should be displayed. Or something like that

### Actual behavior:

A white screen will appear
![image](https://user-images.githubusercontent.com/66670106/104321393-d99a2780-54e3-11eb-9c33-01feb5cbad28.png)

### Server Setup Information:

- Version of Rocket.Chat Server: 3.10.3
- Operating System: CentOS 7
- Deployment Method: tar in gz
- Number of Running Instances: 1
- DB Replicaset Oplog: 1G
- NodeJS Version: v12.20.0
- MongoDB Version: 4.0.22

### Client Setup Information

- Desktop App or Browser Version: Google Chrome 87.0.4280.88, FireFox 84.0.2, Opera 70.0.3728.175
- Operating System: Windows 10

### Additional context

<!-- Add any other context about the problem here. -->

### Relevant logs:

**Opera**
7e5d95910ae7b400b66ac5cc926cbe5c611d169d.js?meteor_js_resource=true:1299 Deprecation warning: use moment.updateLocale(localeName, config) to change an existing locale. moment.defineLocale(localeName, config) should only be used for creating a new locale See http://momentjs.com/guides/#/warnings/define-locale/ for more info.
S @ 7e5d95910ae7b400b66ac5cc926cbe5c611d169d.js?meteor_js_resource=true:1299
7e5d95910ae7b400b66ac5cc926cbe5c611d169d.js?meteor_js_resource=true:1226 Error: Cannot find module '../../../client/account/security/AccountSecurityPage'
    at y (7e5d95910ae7b400b66ac5cc926cbe5c611d169d.js?meteor_js_resource=true:3)
    at d.resolve (7e5d95910ae7b400b66ac5cc926cbe5c611d169d.js?meteor_js_resource=true:3)
    at 7e5d95910ae7b400b66ac5cc926cbe5c611d169d.js?meteor_js_resource=true:3
    at new Promise (<anonymous>)
    at d.prefetch (7e5d95910ae7b400b66ac5cc926cbe5c611d169d.js?meteor_js_resource=true:3)
    at d.b.dynamicImport (7e5d95910ae7b400b66ac5cc926cbe5c611d169d.js?meteor_js_resource=true:1335)
    at 7e5d95910ae7b400b66ac5cc926cbe5c611d169d.js?meteor_js_resource=true:1637
    at ve (7e5d95910ae7b400b66ac5cc926cbe5c611d169d.js?meteor_js_resource=true:1226)
    at ji (7e5d95910ae7b400b66ac5cc926cbe5c611d169d.js?meteor_js_resource=true:1226)
    at Hi (7e5d95910ae7b400b66ac5cc926cbe5c611d169d.js?meteor_js_resource=true:1226)

![image](https://user-images.githubusercontent.com/66670106/104323451-76f65b00-54e6-11eb-835f-28d013ea6df2.png)



<!-- Logs from both SERVER and BROWSER -->
<!-- For more information about collecting logs please see: https://rocket.chat/docs/contributing/reporting-issues#gathering-logs -->

--- a/app/ui-master/client/main.js
+++ b/app/ui-master/client/main.js
@@ -30,7 +30,7 @@ function customScriptsOnLogout() {
 	}
 }
 
-createTemplateForComponent('accountSecurity', () => import('../../../client/account/security/AccountSecurityPage'));
+createTemplateForComponent('accountSecurity', () => import('../../../client/views/account/security/AccountSecurityPage'));
 
 callbacks.add('afterLogoutCleanUp', () => customScriptsOnLogout(), callbacks.priority.LOW, 'custom-script-on-logout');
 
@@ -192,7 +192,7 @@ Template.main.helpers({
 		const user = Meteor.user();
 
 		// User is already using 2fa
-		if (!user || (user.services.totp !== undefined && user.services.totp.enabled)) {
+		if (!user || (user.services.totp !== undefined && user.services.totp.enabled) || (user.services.email2fa !== undefined && user.services.email2fa.enabled)) {
 			return false;
 		}
 

